docker-compose.test.yml

```diff
@@ -14,4 +14,7 @@ services:
     environment:
       TEST_DATABASE_URL: "postgres://postgres:postgres@db/postgres?sslmode=disable"
   db:
-    image: postgres
+    image: postgres:9
```

Comment:
Would be even more reliable to use the sha256 pinning here; but then we wouldn't get patches. If this is for testing, might be better to keep it consistent.

Reply:
fixed